include/oxen/quic/opt.hpp

```diff
@@ -6,121 +6,157 @@
 #include "crypto.hpp"
 #include "types.hpp"
 
-namespace oxen::quic::opt
+namespace oxen::quic
 {
-    using namespace std::chrono_literals;
+    class Endpoint;
 
-    struct max_streams
+    namespace opt
```

Comment:
The extra indentation here is a bit annoying; perhaps this could be:
```C++

namespace oxen::quic { class Endpoint; }
namespace oxen::quic::opt
{
...
}
```
to keep it less annoying.